dg/templates/app_dashboards/loop_base.html

```diff
@@ -31,13 +31,13 @@
 
 <!-- Scripts here -->
 <!-- Placed at the end of file for loading faster -->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-<!-- <script src="{% static 'app_dashboards/js/jquery.min.js' %}"></script> -->
-<script src="{% static 'app_dashboards/js/materialize.js' %}"></script>
+<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script> -->
```

Comment:
since there is 1.12.4 already being used on website, can we use same library instead of 1.11 jquery ?